hooks/pre_commit/20-certs_update.rb

```diff
@@ -0,0 +1,18 @@
+if module_enabled?('certs')
+  if app_value('certs_update_server_ca') && !module_enabled?('katello')
+    fail_and_exit("--certs-update-server-ca needs to be used with katello", 101)
+  end
+
+  ca_file   = param('certs', 'server_ca_cert').value
+  cert_file = param('certs', 'server_cert').value
+  key_file  = param('certs', 'server_key').value
+
+  if !app_value('certs_skip_check') && cert_file.to_s != "" &&
+     (app_value('certs_update_server_ca') || app_value('certs_update_server'))
```

Comment:
I'd argue we can drop this. Our certs check should just work. This also means that if we tighten the checks to catch more problems, users will be notified about this, if they don't use `--certs-update-server`.